nvflare/fuel_opt/utils/pyhocon_loader.py

```diff
@@ -63,16 +64,24 @@ def __init__(self):
         super(PyhoconLoader, self).__init__(ConfigFormat.PYHOCON)
 
     def load_config(self, file_path: str) -> Config:
+        from pyhocon import ConfigTree
```

Comment:
for these lines, do we want to instead do optional import at the top or something?